modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -16,5 +16,41 @@ def get_facility_names(facility_ids)
     def facilities_service
       Lighthouse::Facilities::Client.new
     end
+
+    def address_from_facility(facility)
+      address = facility.address['physical']
+      return nil unless address
+
+      Mobile::V0::AppointmentAddress.new(
```

Comment:
Maybe we rename this `Mobile::V0::Address` now that we're using it across products? We determine the address shape for our models so I think it's safe, and we could always add fields if another product needs it (as you did with id).